builder/dockerfile/instructions/parse.go

```diff
@@ -276,13 +277,23 @@ func parseFrom(req parseRequest) (*Stage, error) {
 	if err := req.flags.Parse(); err != nil {
 		return nil, err
 	}
+
+	SpecPlatform := &specs.Platform{}
+	if flPlatform.Value != "" {
+		m, err := platforms.Parse(flPlatform.Value)
+		if err == nil {
```

Comment:
similar here;

```go
if m, err := platforms.Parse(flPlatform.Value); err == nil {
    sp := m.Spec()
    SpecPlatform = &sp
}
```

Wondering though; should this error be returned instead, or would that become a breaking change?